Header header
string turtle_id
bool is_ready